components/segment-bridge/staging/base/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-
 resources:
-  - segment-bridge.yaml
\ No newline at end of file
+  - ../../base
+  
```

Comment:
nit:  trailing spaces, same comment for other files below

Reply:
fixed